gemfiles/rails6.1.gemfile

```diff
@@ -0,0 +1,4 @@
+eval_gemfile 'common.rb'
+
+gem 'rails', '~> 6.1.0'
+gem 'actionpack-action_caching', github: 'rails/actionpack-action_caching', ref: '7bdfa663274a2620dde8daad7dcb995c1cfef840'
```

Comment:
I wasn't sure what the story was here. It looks like `actionpack-action_caching` has Rails 6.1 support but it hasn't been released yet. 

👍

Reply:
Exactly. I can try to push for them getting a release out before merging this PR.

Reply:
I don't see a reason for it to block this from being merged, I would ship this and reach out to them.

Reply:
I asked in https://github.com/rails/actionpack-action_caching/pull/77#issuecomment-759345436. Let's see if we don't get a new version in a day or two.